src/api/address/handlers/get-addresses.ts

```diff
@@ -0,0 +1,55 @@
+import omit from 'lodash.omit';
+
+import type { BigCommerceAddressesResponse, AddressHandlers } from ".."
+import getCustomerId from "../../operations/get-customer-id"
+
+const getAddresses: AddressHandlers["getAddresses"] = async ({
+	res,
+	body: { customerToken, page = "1" },
+	config,
+}) => {
+	let response: BigCommerceAddressesResponse = {}
+	if (customerToken) {
+		const customerId =
+			customerToken && (await getCustomerId({ customerToken, config }))
+
+		if (!customerId) {
+			// If the customerToken is invalid, then this request is too
+			return res.status(404).json({
+				data: null,
+				errors: [{ message: 'Invalid request' }],
+			})
+		}
+
+		response = await config.storeApiFetch(
+			`/v3/customers/addresses?customer_id:in=${customerId}&page=${parseInt(page, 10)}`,
+			{
+				headers: {
+					Accept: "application/json",
+				},
+			}
+		)
+	}
+
+	if (!response || !response.data || !response.meta?.pagination) {
+		res.status(200).json({ data:  null })
+	} else {
+		res.status(200).json({ data: {
+			addresses: response.data,
+			pagination: {
+				...omit(response.meta.pagination, ['links', 'current_page']),
```

Comment:
No need to change, but just a thought: perhaps we can....omit the use of `omit` by destructuring the `pagination` object?

Something similar to the line below would allow use to remove that particular lodash import (which is being removed by the lodash team anyways in v5): 
```const { links, current_page, ...pagination } = response.meta.pagination;```

Reply:
yeah, that's a great idea
We can add it in next release!